src/FastRoute/src/DBWrapper.cpp

```diff
@@ -1337,6 +1325,19 @@ void DBWrapper::getFixedInstances(r_tree& fixedInsts)
   }
 }
 
+void DBWrapper::createMapForDbNets() {
```

Comment:
you don't need this function. dbBlock::findNet.
but you shouldn't be using net names for anything any more